vendor/zkllvm-metacraft-circuits/src/circuits/CMakeLists.txt

```diff
@@ -3,3 +3,9 @@ add_example(weigh_justification_and_finalization SOURCES weigh_justification_and
 
 add_dependencies(template compute_shuffled_index)
 add_dependencies(template weigh_justification_and_finalization)
+
+add_custom_command(TARGET compute_shuffled_index POST_BUILD
+    COMMAND cp -r ${CMAKE_CURRENT_SOURCE_DIR}/compute_shuffled_index.json ${CMAKE_CURRENT_BINARY_DIR}
+    DEPENDS compile_commands.json
```

Comment:
Check if this dependency is required. POST_BUILD is probably enough

Reply:
It is strange, but it is required by cmake.